ca_on_toronto/committees.py

```diff
@@ -4,8 +4,56 @@
 from .helpers import build_lookup_dict, committees_from_sessions
 from .constants import TWO_LETTER_ORG_CODE_SCHEME
 
+import re
+
+
+MEMBERS_URL_TEMPLATE = 'http://app.toronto.ca/tmmis/decisionBodyProfile.do?function=doGetMembers&meetingId={}&showLink=true'
+DEFAULT_COMMITTEE_ROLE = 'Member'
```

Comment:
Actually, not seeing that in NYC or Chicago data:

* http://ocd.datamade.us/ocd-person/b998bc09-5e16-4117-a93a-e25d41b826a2/
* https://github.com/datamade/chi-councilmatic/blob/master/councilmatic/settings_jurisdiction.py#L98-L102

I'd prefer to stay consistent so I don't have to get title-case changes into django-councilmatic